.circleci/config.yml

```diff
@@ -1,17 +1,95 @@
 version: 2.1
 
 jobs:
-  build_lint_test:
+  nexus:
     docker:
       - image: circleci/node:10
     steps:
       - checkout
-      - run: yarn install
+      - run: yarn --frozen-lockfile
       - run: yarn build
       - run: yarn lint
       - run: yarn test:ci
 
+  example_apollo-fullstack:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn --cwd=examples/apollo-fullstack --frozen-lockfile
+      - run: yarn --cwd=examples/apollo-fullstack tsc
+
+  example_ghost:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn link
+      - run: yarn --cwd=examples/ghost --frozen-lockfile
+      - run: yarn --cwd=examples/ghost link nexus
+      - run: yarn --cwd=examples/ghost tsc
+
+  example_githunt-api:
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn link
+      - run: yarn --cwd=examples/githunt-api --frozen-lockfile
+      - run: yarn --cwd=examples/githunt-api link nexus
+      - run: yarn --cwd=examples/githunt-api tsc
+
+  example_kitchen-sink:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn --cwd=examples/kitchen-sink --frozen-lockfile
+      - run: yarn --cwd=examples/kitchen-sink tsc
+
+  example_nexus-prisma:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn --cwd=examples/nexus-prisma --frozen-lockfile
+      - run: yarn --cwd=examples/nexus-prisma tsc
+
+  example_star-wars:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run: yarn --frozen-lockfile
+      - run: yarn link
+      - run: yarn --cwd=examples/star-wars --frozen-lockfile
+      - run: yarn --cwd=examples/star-wars link nexus
+      - run: yarn --cwd=examples/star-wars tsc
```

Comment:
We should make sure the definition file is generated before running tsc? If you have an idea on how to make this in a clean way @Weakky @jasonkuhrt?

Reply:
I'm thinking about running node `dist/schema.js` and then `git diff src/schema.graphql` and  `git diff src/nexus.d.ts` and check that there is no diff.